x-pack/plugins/security_solution/public/timelines/store/timeline/epic.ts

```diff
@@ -182,8 +132,9 @@ export const createTimelineEpic =
             myEpicTimelineId.setTemplateTimelineVersion(addNewTimeline.templateTimelineVersion);
             return getOr(false, 'payload.savedTimeline', action);
           } else if (
-            timelineActionsType.includes(action.type) &&
-            !timelineObj.isLoading &&
+            action.type === saveTimeline.type &&
+            // don't save again when already saving, unless the timeline changed
+            (!timelineObj.isLoading || timelineObj.changed) &&
```

Comment:
Shouldn't it be 

```javascript
!timelineObj.isSaving
```

Reply:
I think my comment was misleading here. I fixed it in https://github.com/elastic/kibana/pull/169239/commits/8df3e7fa9df271be296b7a77cbde24b26043443c.
@logeekal let me know if it makes more sense.

`isLoading` isn't really tested and so I did not want to change this logic too much to.